doc/src/fix_vector.rst

```diff
@@ -43,6 +45,10 @@ values, they are stored as rows in a global array, whose number of
 rows is growing.  The resulting vector or array can be used by other
 :doc:`output commands <Howto_output>`.
 
+The optional *nmax* keyword can be used to restrict the length of the
+vector to the given *length* value. Once the vector is filled, the
+oldest entries will be discarded when new entries are added.
```

Comment:
Should it be mentioned either here or at the bottom of the page that the default behavior is *nmax* = the max integer  value so users won't expect it to store data indefinitely if *nmax* isn't specified?

Maybe you already described this somewhere in the doc page and I missed it.